js/scripts.js

```diff
@@ -31,19 +31,21 @@ let a24Repository = (function() {
 	// function for adding a movie
 	function add(movie) {
 		if (typeof movie === 'object' && 'name' in movie && 'year' in movie && 'length' in movie && 'genre' in movie) {
-			a24List.push(movie);
+			movieList.push(movie);
 		} else {
 			return 'invalid input';
 		}
 	}
 	// function to show all movies
 	function getAll() {
-		return a24List;
+		return movieList;
 	}
 
 	// function to find a specific movie
 	function findMovie() {
-		a24List.filter((movie) => movie.name === 'Moonlight');
+		let userInput = '';
+		let result = movieList.filter((movie) => movie.name.toLowerCase() === userInput.toLowerCase());
+		return result;
```

Comment:
```
function findMovie(userInput) {
    return movieList.filter(
      (movie) => movie.name.toLowerCase() === userInput.toLowerCase()
    );
  }
  ```
  
  Notice the parameter `useInput`. You will then access it using
  `a24Repository.findMovie("Moonlight")`